tools/generate-icons.js

```diff
@@ -71,14 +71,18 @@ export { ${pascalCase(fileName)} };
 const optimizeSvg = (svg) => {
   const optimized = optimize(svg, {
     multipass: true,
-    plugins: extendDefaultPlugins([
-      // viewBox is required to resize SVGs with CSS.
-      // @see https://github.com/svg/svgo/issues/1128
+    plugins: [
```

Comment:
Changing the plugins here seems unexpected to add an icon. Just confirming this is intentional?

Reply:
Yup. The generate-icons script was broken by a recent upgrade of dependencies. I'm guessing nobody ran a generate-icons since the dependency was updated.

Reply:
Thanks for clarifying!